code/style.css

```diff
@@ -1,5 +1,315 @@
-/* If you can see a peach background, the css file is correctly linked. Feel free to change or remove this styling! */
-
 body {
-  background-color: peachpuff;
+  background-color: rgb(169, 203, 206);
```

Comment:
Using two different stylesheets could be good sometimes, but to avoid having to repeat your code you could create one "main" stylesheet. If your other pages need their own styling, make sure to import the main stylesheet, and put only the different styles in the second file. You can always import more stylesheets. 😊